collect_app/src/main/java/org/odk/collect/android/preferences/GeneralPreferencesFragment.java

```diff
@@ -84,6 +85,7 @@ public boolean onPreferenceClick(Preference preference) {
             getActivity()
                     .getFragmentManager()
                     .beginTransaction()
+                    .setCustomAnimations(R.animator.push_left_in_fragment,R.animator.push_left_out_fragment,R.animator.push_right_in_fragment,R.animator.push_right_out_fragment)
```

Comment:
Add a space after each comma.  Since this line is so long, the four items should also be on separate lines.